### ISSUE TITLE
Urgent Help Needed

### ISSUE THREAD

**Issue by vzelenko** [rails/webpacker#2638]
Is there any compilation log for `bundle exec rake webpacker:compile` command???

I've been getting a `Compilation failed:` message after compilation attempt and had NO IDEA what's causing it.  Even running `webpack-dev-server` gives no errors whatsoever.

I'm stuck in mid-release.

ruby version:  2.3.3
rails version:  4.2.11.1
webpacker:  4.2.2

[UPDATE]
Managed to get by running `./bin/webpack`

**Comment by akoskm** [comment]
Can we close this @vzelenko?

**Comment by vzelenko** [comment]
> Can we close this @vzelenko?

I have worked around the issue, so it is not urgent.  I still want to know if/where there is logging tht would help me troubleshoot these phantom compilation failures.

Thanks.

**Comment by akoskm** [comment]
Are you experiencing something similar to this https://github.com/rails/webpacker/issues/2276#issuecomment-533985274?

**Comment by vzelenko** [comment]
I'm not sure.  As far as I can tell, all the JS compilation works fine.  When I use `webpack-dev-server` or `webpack` to compile JS assets, there is NO ERROR shown and my code appears to work.

When I use `bundle exec rake assets:clobber assets:precompile` and specifically, `webpacker:compile` the issue is NOT displayed in console.

All I want is to be able to understand what is failing on webpacker's side.

**Comment by rossta** [comment]
What is the setting of `webpack_compile_output` in your `config/webpacker.yml`?

**Comment by vzelenko** [comment]
Currently, `false` - but I do believe I've tried `true` and got the same result.

One small update.  I've tried a release today, where I basically exclude webpacker compile, and then running it manually as so:

```bash
  bundle exec rake assets:clobber
  echo "*** LOG *** Precompiling assets"
  WEBPACKER_PRECOMPILE=false bundle exec rake assets:precompile
  echo "*** LOG Compiling Webpack assets ***"
  NODE_ENV=production ./bin/webpack
```

During webpack build I first got a message `Killed` printed to the console, as if the process was killed by something.  Second run was successful (of `NODE_ENV=production ./bin/webpack`.)

Noteworthy?
